manifests/plugin/foreman_gutterball.pp

```diff
@@ -1,12 +1,18 @@
 # foreman_gutterball plugin
 class katello_devel::plugin::foreman_gutterball {
   $path = "${katello_devel::deployment_dir}/foreman-gutterball"
+  $foreman_settings_path = "${katello_devel::deployment_dir}/foreman/config/settings.plugins.d"
```

Comment:
@komidore64 i think i have the right dir for prod:
https://github.com/Katello/puppet-katello/blob/afa0d0f69acfae9a7d27710301671e1a102962d4/manifests/params.pp#L47
that dir then gets copied over to /usr/share/foreman/config/settings.plugins.d/